docs/tutorials/quick-start/index.rst

```diff
@@ -0,0 +1,645 @@
+    fully qualified domain name.
+
+Once it is deployed, you can use the command `kubectl get ingress` to see the status
+ of the ingress:
+
+.. code-block:: shell
+
+    NAME      HOSTS     ADDRESS   PORTS     AGE
+    kuard     *                   80, 443   17s
+
+It may take a few minutes, depending on your service provider, for the ingress to be 
+fully created. When it has been created and linked into place, the ingress will show an
+address as well:
+
+.. code-block:: shell
+
+    NAME      HOSTS     ADDRESS         PORTS     AGE
+    kuard     *         35.199.170.62   80        9m
+
+.. note::
+
+    The IP address on the ingress *may not* match the IP address that the 
+    nginx-ingress-controller. This is fine, and is a quirk/implementation detail 
+    of the service provider hosting your Kubernetes cluster. Since we are using 
+    the nginx-ingress-controller instead of any cloud-provider specific ingress 
+    backend, use the IP address that was defined and allocated for the 
+    nginx-ingress-service LoadBalancer resource as the primary  access point for
+    your service.
+
+Make sure the service is reachable at the domain name you added above, for example
+`http://example.your-domain.com`. The simplest way is to open a browser and enter the
+name that you set up in DNS, and for which we just added the ingress.
+
+You may also use a command line tool like `curl` to check the ingress.
+
+.. code-block:: shell
+
+    $ curl -kivL -H 'Host: example.your-domain.com' 'http://35.199.164.14'
+
+The options on this curl command will provide verbose output, following any redirects,
+show the TLS headers in the output,  and not error on insecure certificates. With 
+nginx-ingress-controller, the service will be available with a TLS certificate, but it
+will be using a self-signed certificate provided as a default from the 
+nginx-ingress-controller. Browsers will show a warning that this is an invalid certificate.
+This is expected and normal, as we haven't yet used cert-manager to get a fully trusted
+certificate for our site.
+
+.. warning::
+
+    It is critical to make sure that your ingress is available and responding correctly 
+    on the internet. This quickstart example uses LetsEncypt to provide the certificates,
+    which expects and validates both that the service is available and that during the
+    process of issuing a certificate uses that valdiation as proof that the request for 
+    the domain belongs to someone with sufficient control over the domain.
+
+Step 3 - Deploy Cert Manager
+============================
+
+We need to install cert-manager to do the work with kubernetes to request a certificate
+and respond to the challenge to validate it. We can use helm to install cert-manager. This
+example installed cert-manager into the `kube-system` namespace from the public helm
+charts.
+
+.. code-block:: shell
+
+    $ helm install --name cert-manager --namespace kube-system stable/cert-manager
```

Comment:
There's no particular reason why we should deploy into `kube-system`. I think we're probably best to advise users deploy into `cert-manager` instead, to keep things clean and tidy 😄

Reply:
👍 Got it - another poorer assumption on my part, I thought it was some permission issue/need. I'll move it into it's own namespace

Reply:
🤔 not sure if this was missed in the update/somehow? This is still `kube-system` (and so is the `NAMESPACE: kube-system` line below from the helm install)

Reply:
erp, fixin